x-pack/plugins/ingest_manager/server/routes/epm/handlers.ts

```diff
@@ -165,23 +166,29 @@ export const installPackageHandler: RequestHandler<TypeOf<
     };
     return response.ok({ body });
   } catch (e) {
+    if (e instanceof PackageNotFound) {
+      return response.notFound({
+        body: { message: `${e.message} not found` },
```

Comment:
If the complete message is constructed and a status is set when the error is thrown, couldn't we check for `instanceof IngestManagerError` here and pass status code and message through to a `customError`?

Also, I think it would be useful to log out every error in the handler too, not only in the catch-all case at the end.